components/dashboard/src/teams/sso/OIDCClients.tsx

```diff
@@ -29,16 +33,44 @@ type OIDCClientsListProps = {
     clientConfigs: OIDCClientConfig[];
 };
 const OIDCClientsList: FC<OIDCClientsListProps> = ({ clientConfigs }) => {
+    const { toast } = useToast();
     const [showCreateModal, setShowCreateModal] = useState(false);
+    const [activateModalConfigId, setActivateModalConfigID] = useState("");
+
+    const verifyClient = useVerifyClient({
+        onSuccess: (configId: string) => {
+            toast(
+                <span>
+                    Single sign-on configuration was successfully verified.{" "}
+                    <LinkButton onClick={() => setActivateModalConfigID(configId)}>Activate configuration</LinkButton>.
+                </span>,
+            );
+        },
+        onError: (errorMessage) => {
+            toast(errorMessage);
+        },
+    });
 
     const onCreate = useCallback(() => setShowCreateModal(true), []);
     const hideModal = useCallback(() => setShowCreateModal(false), []);
 
+    const handleSaved = useCallback(
+        (configId: string) => {
+            toast(
+                <span>
+                    Your SSO configuration was successfully saved.{" "}
+                    <LinkButton onClick={() => verifyClient(configId)}>Verify configuration</LinkButton>.
```

Comment:
**thought**: Didn't notice we already added a `LinkButton` recently. We could add this as a component variant for the `Button` component but looks good also as is for now—the eternal dilemma: _new component_ vs _new variant_.  😃